types/user/hash.pp

```diff
@@ -0,0 +1,3 @@
+type Accounts::User::Hash = Hash[
+  NotUndef[Accounts::User::Name], Accounts::User::Resource
```

Comment:
`NotUndef` is redundant here since `Accounts::User::Name` is a `Pattern` and a pattern does not accept `undef` as a value.

Reply:
Removed.